### PR TITLE
Updated docs to be more up to date (Mostly manjaro related changes)

### PR DESCRIPTION
**Pull request by thepolarbear0818**
The edits were mainly to swap out the older repo links to the new ones. I'm not quite sure how we handle old things here, so I just kinda shoved a "Outdated" thing by it

### CODE REVIEW

**Comment by thepolarbear0818** [comment]
Also removed the part about the kernels on the Manjaro introduction since they were outdated. Replaced with an explanation on why there are 2 different ISO's

File: docs/index.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ If the distribution you want to use has a guide [here](https://wiki.t2linux.org/
 - EndeavourOS [https://github.com/t2linux/EndeavourOS-ISO-t2](https://github.com/t2linux/EndeavourOS-ISO-t2)
 - Fedora [https://github.com/mikeeq/mbp-fedora](https://github.com/mikeeq/mbp-fedora)
 - Gentoo [https://github.com/t2linux/T2-Gentoo-Kernel](https://github.com/t2linux/T2-Gentoo-Kernel)
-- Manjaro [https://github.com/JPyke3/mbp-manjaro](https://github.com/JPyke3/mbp-manjaro)
+- Manjaro (Jpyke3 - Outdated) [https://github.com/JPyke3/mbp-manjaro](https://github.com/JPyke3/mbp-manjaro)
+- Manjaro (NoaHimesaka - Reccomended) [https://github.com/NoaHimesaka1873/manjaroiso-t2](https://github.com/NoaHimesaka1873/manjaroiso-t2)
```
Comment (AdityaGarg8) [inline docs/index.md:37]: It's Recommended

File: docs/roadmap.md [hunk 2]
```diff
@@ -37,8 +37,7 @@ Arch Linux probably has the most documentation, both officially for the whole pr
 t2linux wiki with a really detailed and up to date install guide. On top of that there is also a work in progress `archinstall` script.
 Keep in mind however that the whole process is done in the command line.
 
-Manjaro has a more guided install experience, with a graphical installer and multiple prebuilt ISOs for different desktop environments. Note that Manjaro currently only has 5.7.19 as the latest kernel version. The support period for 5.7.x ended in August 2020.
-You could install the linux-mbp-arch kernel, which is updated, by following step 8 of the [arch install guide](https://wiki.t2linux.org/distributions/arch/installation/), and `sudo pacman -S linux-mbp linux-mbp-headers`.
+Manjaro has a more guided install experience, with a graphical installer and multiple prebuilt ISOs for different desktop environments. There are 2 versions of the Manjaro ISO built by different people. JPyke3's version, which is more outdated and requires more configuration, and NoaHimesaka1873's which is newer and reccomended.
```
Comment (AdityaGarg8) [inline docs/roadmap.md:40]: Again, recommended, not reccomended. Also, Remove the trailing space to fix lint issues.

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

**Review by thepolarbear0818 — COMMENTED** [review]
Seems okay to me, Might want to tell the reader somewhere that there is a new repo, but I think that it will do fine

**Comment by thepolarbear0818** [comment]
btw, I'm very new to the github pull request thing, so I might not completely get how things work around here

File: docs/roadmap.md [hunk 3]
```diff
@@ -37,8 +37,9 @@ Arch Linux probably has the most documentation, both officially for the whole pr
 t2linux wiki with a really detailed and up to date install guide. On top of that there is also a work in progress `archinstall` script.
 Keep in mind however that the whole process is done in the command line.
 
-Manjaro has a more guided install experience, with a graphical installer and multiple prebuilt ISOs for different desktop environments. Note that Manjaro currently only has 5.7.19 as the latest kernel version. The support period for 5.7.x ended in August 2020.
-You could install the linux-mbp-arch kernel, which is updated, by following step 8 of the [arch install guide](https://wiki.t2linux.org/distributions/arch/installation/), and `sudo pacman -S linux-mbp linux-mbp-headers`.
+Manjaro has a more guided install experience, with a graphical installer and multiple prebuilt ISOs for different desktop environments. There are 2 versions of the Manjaro ISO built by different people. JPyke3's version, which is not reccomended, and NoaHimesaka1873's which is newer and reccomended.
```
Comment (AdityaGarg8) [inline docs/roadmap.md:40]: It's Recommended, not Reccomended. Please correct this typo

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/roadmap.md (hunk 3 above)
Comment (thepolarbear0818) [reply docs/roadmap.md]: woops, missed the typo there

**Review by thepolarbear0818 — COMMENTED** [review]
Fixed the typo's now (or at least i think)

**Comment by AdityaGarg8** [comment]
@NoaHimesaka1873, could you review this please.

**Review by NoaHimesaka1873 — APPROVED** [review]
Seems okay.

File: docs/roadmap.md [hunk 4]
```diff
@@ -37,8 +37,9 @@ Arch Linux probably has the most documentation, both officially for the whole pr
 t2linux wiki with a really detailed and up to date install guide. On top of that there is also a work in progress `archinstall` script.
 Keep in mind however that the whole process is done in the command line.
 
-Manjaro has a more guided install experience, with a graphical installer and multiple prebuilt ISOs for different desktop environments. Note that Manjaro currently only has 5.7.19 as the latest kernel version. The support period for 5.7.x ended in August 2020.
-You could install the linux-mbp-arch kernel, which is updated, by following step 8 of the [arch install guide](https://wiki.t2linux.org/distributions/arch/installation/), and `sudo pacman -S linux-mbp linux-mbp-headers`.
+Manjaro has a more guided install experience, with a graphical installer and multiple prebuilt ISOs for different desktop environments. There are 2 versions of the Manjaro ISO built by different people. JPyke3's version, which is not recommended, and NoaHimesaka1873's which is newer and recommended.
+
+EndeavourOS is an Arch based Linux distribution with a graphical installer. This distribution requires little to no configuration after install.
```
Comment (AdityaGarg8) [inline docs/roadmap.md:42]: There is a space at the end of EndeavourOS Line. Remove it to fix the Lint error

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/roadmap.md (hunk 4 above)
Comment (AdityaGarg8) [reply docs/roadmap.md:42]: Nvm, I’ll fix it.

**Pull request merged**